apiary.apib

```diff
@@ -123,57 +123,83 @@ switching a flag in the Chrome extension.
 | agent_can_request_control | false | Can the agent request control after which it can be granted |
 | agent_can_take_control | false | Can the agent take control without requirement that the controller needs to grant it |
 | agent_can_end_session | true | Show End Session button on follower's end |
-| filesharing | true | Allow file sharing |
 | allow_original_file_download | false | Allow users to download shared files (when set to false, users can only view them) |
-| store_chat_logs | false | If enabled, chat logs will be available for download in Surfly dashboard |
-| on_end_redirect_follower_to_queue | false | After session end return the follower to the Surfly Queue page |
-| leader_redirect_url | false | After session end redirect the leader to a custom URL |
+| blacklist | "[]" | (enterprise only) Restrict access to the specific resources. [More...](#restrictions)|
+| enable_sounds | true | Enable sound notifications |
+| filesharing | true | Allow file sharing |
 | follower_redirect_url | false | After session end redirect the follower to a custom URL |
 | format_session_id | true | Change the session id in something easy to communicate over the phone (eg, 123-123-123) |
-| videochat | true | Is videochat allowed |
-| splash | true | Do we need to show the splash screen on session start |
+| leader_redirect_url | false | After session end redirect the leader to a custom URL |
+| low_quality_video | false | Video quality is limited to 320x240@15 |
 | newurl  | true | Should we show the 'newurl button' in the widget |
+| on_end_redirect_follower_to_queue | false | After session end return the follower to the Surfly Queue page |
 | sharing_button | true | Should we show the 'sharing button' in the widget |
-| enable_sounds | true | Enable sound notifications |
+| splash | true | Do we need to show the splash screen on session start |
 | start_muted | false | All participants start with muted microphone|
-| low_quality_video | false | Video quality is limited to 320x240@15 |
+| store_chat_logs | false | If enabled, chat logs will be available for download in Surfly dashboard |
+| videochat | true | Is videochat allowed |
+| whitelist | "[]" | (enterprise only) Allow access only to the specific resources. [More...](#restrictions)|
+
+<a name="restrictions"></a>
+#### Restrictions format
+A string representation of the JSON array is expected. Every element of the array is an object with the following properties:
+ - `pattern` - regular expresstion. Requested url will be matched to this regular expression
+ - (optional) `redirect` - an url to redirect user to
+
+##### Blacklist example:
+Restrict access to the `example.com`. When user tries to access `example.com` he will be redirected to the default page provided by Surfly
+
+```
+{
+    "pattern": ".*example\\.com.*"
+}
+```
 
+##### Whitelist example:
+Allow access only to the `example.com`. When user tries to access `example.com` he will be redirected to the provided redirect url
```

Comment:
shouldn't this be "When user tries to access anything other other than `example.com`".